doc/ref/configuration/minion.rst

```diff
@@ -583,11 +583,11 @@ Pull port used when :conf_minion:`ipc_mode` is set to ``tcp``.
 ``transport``
 -------------
 
-Default: ``zeromq``
+Default: ``ZeroMQ``
```

Comment:
Same thing in this file. Code settings should still be ``zeromq``, but used in a sentence should be ``ZeroMQ``.